Makefile

```diff
@@ -1,9 +1,10 @@
 NAME ?= percona-xtradb-cluster-operator
 IMAGE_TAG_OWNER ?= perconalab
 IMAGE_TAG_BASE ?= $(IMAGE_TAG_OWNER)/$(NAME)
-VERSION ?= $(shell git rev-parse --abbrev-ref HEAD | sed -e 's^/^-^g; s^[.]^-^g;' | tr '[:upper:]' '[:lower:]')
+VERSION ?= $(shell git rev-parse --abbrev-ref HEAD | $(SED) -e 's^/^-^g; s^[.]^-^g;' | tr '[:upper:]' '[:lower:]')
 IMAGE ?= $(IMAGE_TAG_BASE):$(VERSION)
 DEPLOYDIR = ./deploy
+SED := $(shell which gsed || which sed)
```

Comment:
maybe we need to move it before VERSION?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1724/commits/95c26a2c4b0fed99e50f66d2aeba66f92aa400d7